### DISCUSSION TITLE
make pre-filled values on entity settings  'selectable' first off

### DISCUSSION THREAD

**Discussion by Mariusthvdb** [home-assistant/frontend#11296] [Other feature requests]
## The request

When editing an entity in the UI the field for Entity ID is selectable, allowing for easy fixing of the desired text, without having to completely re-fill out the field.

However, the fields for Name Override and Icon Override are not selectable, though the pre-filled text is given. If one wants to edit this field, the full field needs to be re-written. Even if only a single character or part of the pre-filled text would need to be changed

## Additional information
<img width="516" alt="Schermafbeelding 2020-09-22 om 23 01 02" src="https://user-images.githubusercontent.com/33354141/93937945-d60e4500-fd28-11ea-9558-b550571411ec.png">

originally the Entity Id was
```
binary_sensor.hvb_macbook_pro_hvb_macbook_pro_microfoon
```
as still reflected in the Name Override field. Entity_id was a simple select/cut procedure. Name Override needs a full retype. 
Btw, after one has edited that, one Can select/edit.

**Comment by cnotin** [comment]
Here's a PR to hopefully fix that: https://github.com/home-assistant/frontend/pull/51894
